pittapi/library.py

```diff
@@ -162,4 +161,4 @@ def reserved_hillman_times():
         }
         for reservation in data
     ]
-    return bookings
+    return bookings
```

Comment:
Please make sure that all files end with an empty line (we check for this as part of our pre-commit)